mixer/adapter/kubernetesenv/kubernetesenv.go

```diff
@@ -95,8 +96,9 @@ var _ ktmpl.HandlerBuilder = &builder{}
 
 // GetInfo returns the Info associated with this adapter implementation.
 func GetInfo() adapter.Info {
+	singletonBuilder := newBuilder(newKubernetesClient)
```

Comment:
Doesn't this initialize a newBuilder with each `GetInfo` call?

Reply:
GetInfo is only called during registration